pkgs/by-name/cp/cppman/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  groff,
+  nix-update-script,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "cppman";
+  version = "0.5.9";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "aitjcize";
+    repo = "cppman";
+    tag = version;
+    hash = "sha256-iPJR4XAjNrBhFHZVOATPi3WwTC1/Y6HK3qmKLqbaK98=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+    wheel
+    distutils
+  ];
+
+  dependencies = [
+    python3Packages.beautifulsoup4
+    python3Packages.html5lib
+    python3Packages.lxml
+    python3Packages.six
+    python3Packages.soupsieve
+    python3Packages.typing-extensions
+    python3Packages.webencodings
+    groff
+  ];
+
+  # cppman pins all dependency versions via requirements.txt as install_requires, causing this check to fail.
+  dontCheckRuntimeDeps = true;
+
+  pythonImportsCheck = [
+    "cppman"
+  ];
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
Can a `checkPhase` be added? Looks like it is run in CI.

https://github.com/aitjcize/cppman/blob/77b48da5f52d4105e2bc6a860f2c5cfaeadda01a/.github/workflows/test.yml#L30-L32